v2/internal/controllers/crd_networking_privatednszone_test.go

```diff
@@ -48,3 +53,48 @@ func Test_Networking_PrivateDnsZone_CRUD(t *testing.T) {
 	tc.Expect(retryAfter).To(BeZero())
 	tc.Expect(exists).To(BeFalse())
 }
+
+func VirtualNetworkLinks_CRUD(tc *testcommon.KubePerTestContext, zone *network.PrivateDnsZone, rg *resources.ResourceGroup) {
+	vnet := newVMVirtualNetwork(tc, testcommon.AsOwner(rg))
+	tc.CreateResourceAndWait(vnet)
+
+	links := newVirtualNetworkLink(tc, zone, vnet)
+
+	tc.CreateResourcesAndWait(links)
```

Comment:
Can we make the link and the vnet at the same time? So:

```
vnet := newVMVirtualNetwork(tc, testcommon.AsOwner(rg))
links := newVirtualNetworkLink(tc, zone, vnet)
tc.CreateResourcesAndWait(vnet, links)
```

Should be able to, right?

Reply:
If we can't, we have a problem, because users will definitely want to put them into the same YAML file and have ASO work out the sequencing.